mantle/kola/harness.go

```diff
@@ -1314,6 +1329,7 @@ func makeNonExclusiveTest(bucket int, tests []*register.Test, flight platform.Fl
 // analysis after the test run. It should already exist.
 func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flight) {
 	h.Parallel()
+	h.SetSubtests(t.Subtests)
```

Comment:
This will also make sure that `report.json` has the correct subtests for nonexclusive tests?

Reply:
Yes techinically, after we setSubtests, the `report.json` will update with the right subtests for each test. If you are asking whether we are testing to check if the subtests `exists` AND are `correct,` then the answer is no.